rdt/hyper_transformer.py

```diff
@@ -603,8 +603,8 @@ def fit(self, data):
         self._validate_detect_config_called(data)
         self._unfit()
         self._input_columns = list(data.columns)
-        for field in self._input_columns:
-            data = self._fit_field_transformer(data, field, self.field_transformers[field])
+        for field_column, field_transformer in self.field_transformers.items():
+            data = self._fit_field_transformer(data, field_column, field_transformer)
```

Comment:
I don't think this is going to work alone. The `_fit_field_transformer` method doesn't know to pass the columns as a dict in the multi column case

Reply:
In the multi column case we pass the columns as a tuple or as a dict? I supposed it was a tuple and so it was good for me based on the docstring of `_fit_field_transformer(self, data, field, transformer)`:
```
field (str or tuple):
                Name of column or tuple of columns in data that will be transformed
```

Reply:
In the base class you made fit takes in data and a dictionary. It needs to be a dict otherwise the values will always have to be passed in a specific order

Reply:
@R-Palazzo I think this is still the case. The [issue](https://github.com/sdv-dev/RDT/issues/683) states that the `HyperTransformer` should run with a multi-column transformer too.